### ISSUE TITLE
autocompletion for names in console

### ISSUE THREAD

**Issue by def-** [ddnet/ddnet#1489]
For example to use add_foe it would be convenient if the auto-completion would cycle through the player names. We should not just add them to the list of commands but instead recognize when a player name is the next argument.

https://forum.ddnet.tw/viewtopic.php?t=6777

**Comment by fokkonaut** [comment]
open chat, autocomplete name, ctrl+c, f1, ctrl+v

**Comment by Learath2** [comment]
Could use victim and then parse the command params in `CGameConsole`

**Comment by fokkonaut** [comment]
@Learath2, what about commands that use r[name/text]? Would also be good if those can have autocompletion

**Comment by Learath2** [comment]
If we add completion to victim parameters then we can rework all the ones
who use r[name] to v

**Comment by fokkonaut** [comment]
I dont think that is a good idea, victim cant handle names with spaces if they dont use quotes.
